docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -266,8 +266,11 @@ The master supports the following configuration settings:
          Determined master.
 
       -  ``slot_type``: The default slot type assumed when users request resources from Determined
-         in terms of ``slots``. Defaults to ``cuda`` for partitions where GPUs are detected
-         automatically, else ``cpu``. If GPUs cannot be detected automatically, for example when
+         in terms of ``slots``. Available values are ``cuda``, ``rocm`` and ``cpu``, where 1
+         ``cuda`` or ``rocm`` slot is 1 GPU, otherwise CPU slots are requested. The number of CPUs
```

Comment:
Available values are ``cuda``, ``rocm`` and ``cpu``, where 1 ``cuda`` or ``rocm`` slot is 1 GPU. Otherwise, CPU slots are requested.

(reason: two independent clauses)

Reply:
Taken care if in this PR:

https://github.com/determined-ai/determined/pull/5918